uk.co.compucorp.civicrm.civihrsampledata/README.md

```diff
@@ -0,0 +1,25 @@
+## Intro
+CiviHR Sample Data is an extensions that creates a sample data for CiviHR which
```

Comment:
CiviHR Sample Data is an extension**~~s~~** that creates **~~a~~** sample data for CiviHR